.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,40 @@
+name: 'tests'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+      fail-fast: false
+
+    steps:
+      - name: 'Checkout Project'
+        uses: 'actions/checkout@v2'
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: 'actions/setup-node@v2.1.5'
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: 'Cache Node dependencies'
+        uses: 'actions/cache@v2.1.4'
+        with:
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: ${{ runner.os }}-node-
```

Comment:
Does this decrease the time of the next step? I don't like that I have to dig into some documentation just to figure out what the meaning of the `key` is and what the value of it here does. So I didn't. How much difference does this make?

Reply:
Basically, it allows to cache `node_modules`, so yes it saves some time, I think it is better, so we have a faster CI.

Reply:
Actually, we don't need it. It makes things simpler, if we really want to use cache, we should use `package-lock.json` but that would be for another PR, no need for this one. :+1: